Riot/Modules/Common/Avatar/AvatarService.swift

```diff
@@ -40,14 +40,22 @@ class AvatarService: AvatarServiceType {
         static let avatarDownloadSize = AvatarSize.xxLarge.size
     }
     
-    let avatarGenerator: AvatarGenerator
-    let mediaManager: MXMediaManager
+    private let avatarGenerator: AvatarGenerator
+    private let mediaManager: MXMediaManager
     
     init(avatarGenerator: AvatarGenerator, mediaManager: MXMediaManager) {
         self.avatarGenerator = avatarGenerator
         self.mediaManager = mediaManager
     }
     
+    /**
+     Given an avatar input, this function returns a publisher of UIImage.
+     If possible to retrieve the actualy image it will from network or cache.
```

Comment:
Suggestion:
If possible it will retrieve the actual image from the network or cache.